releng/com.espressif.idf.update/category.xml

```diff
@@ -84,6 +84,9 @@
    <bundle id="org.apache.batik.util">
       <category name="com.espressif.idf.dependencies"/>
    </bundle>
+   <bundle id="org.eclipse.mylyn.wikitext.markdown">
+      <category name="com.espressif.idf"/>
+   </bundle>
```

Comment:
Do we need this? I think we removed it earlier due to some issues or because it was not relevant.”

Reply:
Hi, @kolipakakondal, Isn't this responsible for the markdown editor?

Reply:
We are depending on the commonmark jar for markdown viewer in the new project wizard templates markdown editor
https://github.com/espressif/idf-eclipse-plugin/blob/b52199a64b6b042ab73d1106a1ab8674f1d71354/bundles/com.espressif.idf.ui/META-INF/MANIFEST.MF#L61

Reply:
btw, we need to update the commonmark version to the latest as well. Latest one is https://mvnrepository.com/artifact/org.commonmark/commonmark/0.21.0

Reply:
Hi @kolipakakondal, the common-mark is actually updated to the 0.24 in this PR, but it's used only in the new project page, right? I think the other md files will be opened with wikitext. markdown